styles/components/_top-collections.scss

```diff
@@ -45,8 +45,31 @@
   min-width: 98px;
 }
 
+.no-wrap {
+  white-space: nowrap;
+}
 
 
+.display-above-small {
+  display: flex;
+}
+.display-below-small {
+  display: none;
+}
+
+@include until(540px) {
+  .display-above-small {
+    display: none;
+  }
```

Comment:
If this could help: https://bulma.io/documentation/helpers/visibility-helpers/#hide